src/components/ReportActionItem/ChronosOOOListActions.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {View} from 'react-native';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import reportActionPropTypes from '../../pages/home/report/reportActionPropTypes';
+import styles from '../../styles/styles';
+import Text from '../Text';
+import Button from '../Button';
+import * as Chronos from '../../libs/actions/Chronos';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import DateUtils from '../../libs/DateUtils';
+import OfflineWithFeedback from '../OfflineWithFeedback';
+
+const propTypes = {
+    /** The ID of the report */
+    reportID: PropTypes.string.isRequired,
+
+    /** All the data of the action */
+    action: PropTypes.shape(reportActionPropTypes).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const ChronosOOOListActions = (props) => {
+    const events = lodashGet(props.action, 'originalMessage.events', []);
+
+    if (!events.length) {
+        return (
+            <View style={[styles.flexRow, styles.alignItemsCenter, styles.pt, styles.ml18]}>
+                <Text>You haven&apos;t created any events</Text>
+            </View>
+        );
+    }
+
+    return (
+        <OfflineWithFeedback
+            pendingAction={lodashGet(props.action, 'pendingAction', null)}
+        >
+            <View>
+                {_.map(events, (event) => {
+                    const start = DateUtils.getLocalMomentFromDatetime(props.preferredLocale, event.start.date);
+                    const end = DateUtils.getLocalMomentFromDatetime(props.preferredLocale, event.end.date);
+                    return (
+                        <View key={event.id} style={[styles.flexRow, styles.alignItemsCenter, styles.pt, styles.ml18]}>
+                            {event.lengthInDays > 0 ? (
+                                <Text>
+                                    {event.summary}
+                                    {' for '}
+                                    {event.lengthInDays}
+                                    {event.lengthInDays === 1 ? ' day' : 'days'}
+                                    {' until '}
+                                    {end.format('dddd MMM Do, YYYY')}
+                                </Text>
+                            ) : (
+                                <Text>
+                                    {event.summary}
+                                    {' from '}
+                                    {start.format('h:mma')}
+                                    {' - '}
+                                    {end.format('h:mma')}
+                                    {' on '}
+                                    {start.format('dddd MMM Do, YYYY')}
+                                </Text>
+                            )}
+                            <Button
```

Comment:
I'm not sure about the process here, but should not we disable the button if we have a `pendingAction`?

Reply:
Hm, I don't think it is necessary to. I kind of want to test that out, but I think the idea is that you should be able to delete multiple events while being offline, and that would be OK to leave the button enabled then.

Reply:
The button is per event, why would you leave the button enabled if you pressed it already?

Reply:
Oh, I see what you mean. The specific event is removed in the optimistic data, so the row won't be rendered at all (thus, no button to disable).